src/signals/incident-management/containers/LocationForm/index.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { createStructuredSelector } from 'reselect';
+import { bindActionCreators } from 'redux';
+
+import { patchIncident } from 'models/incident/actions';
+import makeSelectIncidentModel from 'models/incident/selectors';
+
+import Form from './components/Form';
+
+import './style.scss';
```

Comment:
Dit bestand is leeg. Kan deze import verwijderd worden?

Reply:
weggehaald